### PR TITLE
Fixed coding standards, compliance to PSR-2

### PR DESCRIPTION
**Pull request by tristanbes**

### CODE REVIEW

**Comment by sct** [comment]
I am uncomfortable accepting a pull request that is 100% changes that just switches tabs to spaces.

Is there anything else you changed? I can fix this myself with a quick commit. I just want to know what else you may have noticed that isn't compliant without me having to read through your entire commit.

**Comment by tristanbes** [comment]
Haven't you heard of PSR-2 compliance (http://www.php-fig.org/psr/psr-2/) ? I've fixed equals alignement, brackets for controls structure, I translated tabs into spaces and various other coding standards level. 

Why fix it yourself if I already took time to fix it ? Is not that the purpose of github and code collaboration ?

**Comment by tristanbes** [comment]
PS: You shouldn't be unconfortable accepting ANY valuable PR including ANY improvements or fix.

I can find you hundreds of accepted PR on popular open source frameworks that is based only on filemode changes, space problem or any other CS violations.

**Comment by sct** [comment]
I am probably just being overly selfish about the contribution amounts on Github. This is a 100% contribution PR for just compliance. I already fixed it myself locally.

I do understand you are just trying to help but this overall was just a mistake I made in configuring the project in Sublime Text. (As well as missing brackets on the one control structure)

I am pushing the changes now. If you still see anything wrong with the project (alignment issues, missing brackets) feel free to submit another PR and I will accept it.

Again, this isn't about me being against PRs. It's just me being against a PR that is 100% contribution when all you did was change tabs to spaces. I hope you understand.

**Comment by tristanbes** [comment]
You fixed one among other CS problem :-1: . Your code is far away from being PSR-2 valid.

If you're not willing to accept PR because you're afraid to loose some kind of virtual fame, then you should consider setting the repo to read only mode and disabling PR features.

I wanted to use your library in order to make a Symfony2 Bundle, but I won't use it if it's not PSR-2 compliant. 

I took time to make the PR, and i won't take more time to explain to you any rules composed by PSR-2.

PS: Having a repository which have contributors is more rewarding than "I made it myself alone".

You closing my PR : giphy

**Comment by sct** [comment]
I really don't see the differences between the two? Maybe I am making some sort of huge mistake here? That's possible.

I am willing to swallow my pride here and ask you what the major problems are with my compliance.

I will accept your PR and we can go from there. In the end I would rather learn from this than just blow you off.

File: src/sct/League/Champion.php [hunk 1]
```diff
@@ -2,202 +2,207 @@
 
 namespace sct\League;
 
-class Champion {
```
Comment (tristanbes) [inline src/sct/League/Champion.php:5]: Brackets after class declaration must be on a new line

File: src/sct/League/Champion.php [hunk 2]
```diff
@@ -2,202 +2,207 @@
...
-
-	/**
-	 * Create a champion instance using an ID and Name
-	 * 
-	 * @param integer $id   Champion ID
-	 * @param string $name Champion Name
-	 */
-	public function __construct($id, $name)
-	{
-		$this->id = $id;
```
Comment (tristanbes) [inline src/sct/League/Champion.php:63]: If possible (not mandatory but greatly recommended) the "=" must be aligned together. If you are using sublime text 2/3 there's a plugin for that called "Alignement"

File: src/sct/League/Champion.php [hunk 3]
```diff
@@ -2,202 +2,207 @@
...
-	 * Set the stat ranks for this champion
-	 * 
-	 * @param integer $attackRank     Attack Rank
-	 * @param integer $defenseRank    Defense Rank
-	 * @param integer $magicRank      Magic Rank
-	 * @param integer $difficultyRank Difficulty Rank
-	 */
-	public function setRanks($attackRank, $defenseRank, $magicRank, $difficultyRank)
-	{
-		$this->ranks = array("attackRank" => $attackRank, "defenseRank" => $defenseRank,
```
Comment (tristanbes) [inline src/sct/League/Champion.php:165]: Array declaration must be :

``` php
$array = array(
    'key'  => value,
    'key2' => value
    ....
);
```

File: src/sct/League/Champion.php [hunk 4]
```diff
@@ -2,202 +2,207 @@
...
-	 * Returns the boolean indicating if this champion is available in ranked play
-	 * 
-	 * @return boolean
-	 */
-	public function getRankedPlayEnabled()
-	{
-		return $this->rankedPlayEnabled;
-	}
-
-}
```
Comment (tristanbes) [inline src/sct/League/Champion.php:203]: There must be a blank line after the end of each file

File: src/sct/League/Champions.php [hunk 5]
```diff
@@ -4,86 +4,91 @@
...
-	 * Request a specific champion from the champions array. Throws an exception
-	 * if it does not exist.
-	 * 
-	 * @param  string $name Champion Name
-	 * 
-	 * @return array       Champion Array
-	 */
-	public static function getChampion($name)
-	{
-		if (!array_key_exists($name, self::$champions))
```
Comment (tristanbes) [inline src/sct/League/Champions.php:74]: bracket for if structure must be present

File: src/sct/League/League.php [hunk 6]
```diff
@@ -4,164 +4,170 @@
...
-	 * Request a Summoner object by Summoner name
-	 * 
-	 * @param  string $name Summoner Name
-	 * 
-	 * @return Array       Summoner Array
-	 */
-	public function getSummonerByName($name)
-	{
-		$request = $this->client->get('summoner/by-name/' . $name . "?api_key=" . $this->key);
-		return $request->send()->json();
```
Comment (tristanbes) [inline src/sct/League/League.php:87]: Before any `return` statements there must be a blank line (this rules does not apply if the return statement is the only code from a given function)

File: src/sct/League/League.php [hunk 7]
```diff
@@ -4,164 +4,170 @@
...
-	 * Request Summoner Stats by Summoner Name
-	 * 
-	 * @param  string $name Summoner Name
-	 * @param  string $type Type of Request. summary/ranked
-	 * 
-	 * @return Array       Array of requested data
-	 */
-	public function getSummonerStatsByName($name, $type = "summary")
-	{
-		return $this->getSummonerStats($this->getSummonerIdByName($name), $type);
```
Comment (tristanbes) [inline src/sct/League/League.php:114]: example of when you shouldn't use the rule I told in this comment : https://github.com/sct/League/pull/1/files#r8276084

File: src/sct/League/Summoner.php [hunk 8]
```diff
@@ -5,219 +5,223 @@
...
-	 * @param  integer $name Champion ID
-	 * 
-	 * @return Array       Array of Champions ranked stats
-	 */
-	public function getStatsForChampion($id)
-	{
-		$stats = $this->getRankedStats();
-
-		foreach ($stats['champions'] as $champion) {
-			if (isset($champion['id']) && $champion['id'] == $id) {
```
Comment (tristanbes) [inline src/sct/League/Summoner.php:162]: Indentation must be 4 spaces using space (and not tabs)

**Comment by sct** [comment]
Thank you for taking the time to point these things out. I feel pretty stupid for missing the class declaration bracket on a new line.

I did add new lines at the end of my files and switch to spaces instead of tabs. I also know about brackets for control statements.

**Comment by tristanbes** [comment]
Thanks. I'll sleep better tonight. ;)

**Pull request merged**